### PR TITLE
Fixed #641 - Printer manager crashes when console is opened with offline Printers due to spam

### LINKED ISSUE #641: Alot of offline printers will crash printer manager console if it's never called the plugin update (dev)
### Description of the problem

```bash
As titled really, gather it's from #611 changes. Easy fix.
```

### Reproduction of the problem

```bash
Have a load of offline printers and load the printer manager page.
```

### PR DESCRIPTION
**Pull request by NotExpectedYet**

### CODE REVIEW

**Comment by davidzwa** [comment]
Options:
- cache the releases on octofarm.net (although this site resolves quite slow in NL right now)
- cache the releases locally and fall back to hard-coded value to make sure 'something' is known

**Comment by NotExpectedYet** [comment]
> Options:
> 
>     * cache the releases on octofarm.net as public API endpoint
> 
>     * cache the releases locally and fall back to hard-coded value to make sure 'something' is known

This ain't related to this issue?

File: client_src/js/printerManagementRunner.js [hunk 1]
```diff
@@ -3009,7 +3014,12 @@ class dashUpdate {
           }
         }
       } catch (e) {
-        console.log(e);
+        //TODO: Rework this file, this is dangerous and stops any error messages been able to be read without crashing the console...
+        // patched for now so only triggeres once encase we do encounter errors...
+        if (!sseErrorMessageTriggered) {
```
Comment (davidzwa) [inline client_src/js/printerManagementRunner.js:3019]: Nice solution, could you create an issue for cleaning this up? Seems important to be fixed

**Review by davidzwa — APPROVED** [review]

**Comment by davidzwa** [comment]
Didnt test.

File: client_src/js/printerManagementRunner.js (hunk 1 above)
Comment (NotExpectedYet) [reply client_src/js/printerManagementRunner.js:3019]: Yeah, the file needs a similar clean up as settings got tbh, will do

**Comment by NotExpectedYet** [comment]
@davidzwa #612 was already created a bit ago btw :)

**Comment by davidzwa** [comment]
Ok lot of work to finish. Hope to do the same as dashboard in webpack branch.
The dashboard runner is so clean now haha

**Pull request merged**